modules/popup-stack/lib/PopupStack.ts

```diff
@@ -344,7 +344,10 @@ export const PopupStack = {
    * opt-in to toggling. Otherwise there is no way to opt-out of toggle behavior (because the target
    * is not inside `element`).
    */
-  contains(element: HTMLElement, eventTarget: HTMLElement): boolean {
+  contains(element: HTMLElement | null, eventTarget: Element | null): boolean {
+    if (!element || !eventTarget) {
+      return false;
+    }
```

Comment:
I don't think we should change the PopupStack. The PopupStack is meant to be global and shared. This means that your application is using a newer version of the PopupStack, but another application is using an older version of the PopupStack and has registered it first. In this scenario, this behavior will break.